src/controllers/dynakube/token/tokens_test.go

```diff
@@ -161,3 +162,59 @@ func testVerifyTokenValues(t *testing.T) {
 	assert.NoError(t, validTokens.VerifyValues())
 	assert.EqualError(t, invalidTokens.VerifyValues(), "value of token 'whitespaces' contains whitespaces at the beginning or end of the value")
 }
+
+func TestConcatErrors(t *testing.T) {
```

Comment:
Nitpick: looks like it could be a table test.

Reply:
agree, done